dashboard/src/components/FilterList/FilterList.tsx

```diff
@@ -97,7 +101,8 @@ const FilterList = ({
   }
 
   return (
-    <div className="flex flex-wrap gap-4">
+    <div className="flex flex-wrap items-center gap-4">
+      {isLoading && <LoadingCircle />}
```

Comment:
Let's put the `LoadingCircle` at the end of the list, that way the items doesn't move when the loading appears or disappears.

Reply:
another solution is isLoading to be a prop for `LoadingCircle`  and use the visibility property https://tailwindcss.com/docs/visibility

Reply:
I'll put it at the end of the list because making it invisible would create too big of a padding before the filters